schema/pet-insurance.json

```diff
@@ -538,27 +1312,27 @@
         {
             "code": "RegistrationNumberInvalid",
             "category": "client",
-            "description": "Registration number is not accepted by the website"
+            "description": "Registration number is not accepted by the website."
         },
         {
             "code": "RegistrationNumberRequired",
             "category": "client",
-            "description": "Registration number is required by the website"
+            "description": "Registration number is required by the website."
         },
         {
-            "code": "DirectDebitRejected",
+            "code": "PolicyAlreadyPurchased",
             "category": "client",
-            "description": "Direct debit details provided have been rejected by the website."
+            "description": "The policy has already been purchased."
         },
         {
-            "code": "PolicyAlreadyPurchased",
+            "code": "DirectDebitRejected",
             "category": "client",
-            "description": "The policy has already been purchased."
+            "description": "Direct debit details provided have been rejected by the website."
         },
         {
             "code": "BreedTypeRequired",
             "category": "client",
-            "description": "Breed type is required by the website"
+            "description": "Breed type is required by the website."
         }
     ]
 }
```

Comment:
Newline at the end of the line.

I believe both Boris and I have asked for this before. Please configure your editor so that it ends the last line with a newline character.

Github highlights missing newlines at the end with this red character seen above.